kinetic/distribution.yaml

```diff
@@ -4552,6 +4552,21 @@ repositories:
       type: git
       url: https://github.com/KITrobotics/ipr_extern.git
       version: kinetic-devel
+    release:
+      packages:
+      - ipr_extern
+      - libmodbus
+      - libreflexxestype2
+      - ros_reflexxes
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/KITrobotics/ipr_extern-release.git
+      version: 0.8.7-0
+    source:
+      type: git
+      url: https://github.com/KITrobotics/ipr_extern.git
+      version: '''0.8.8'''
```

Comment:
This needs to be a branch name. It looks like it should be `kinetic-devel`

Reply:
thanks! done